Source/Core/Common/Common.vcxproj

```diff
@@ -123,6 +123,7 @@
     <ClInclude Include="GL\GLUtil.h" />
     <ClInclude Include="Hash.h" />
     <ClInclude Include="HttpRequest.h" />
+    <ClInclude Include="Image.h" />
```

Comment:
What is the reasoning behind a broad "image" moniker? Will it house more than just PNG in the future?

Reply:
Possibly, though I don't have any specific plans for additions to the file. If people would prefer to call it something with PNG in the name, I would be fine with that. I'm guessing that linkmauve named it Image because it contained a struct called Image which wasn't PNG-specific, but I've removed that struct in this PR, so that reason doesn't really remain.